src/Core/Addon/Module/ModuleManager.php

```diff
@@ -39,6 +39,7 @@
 use Symfony\Component\EventDispatcher\EventDispatcherInterface;
 use Symfony\Component\HttpFoundation\ParameterBag;
 use Symfony\Component\Translation\TranslatorInterface;
+use Tools;
```

Comment:
Rule no. 1 of migration: do not use legacy classes directly

Reply:
well spotted :)

I'm pretty sure we could "lint" this expectation at some point :+1: